dlgr/griduniverse/experiment.py

```diff
@@ -1452,9 +1495,9 @@ def handle_plant_food(self, msg):
         player = self.grid.players[msg['player_id']]
         position = msg['position']
         can_afford = player.score >= self.grid.food_planting_cost
-        if (can_afford and not self.grid.has_food(position)):
+        if (can_afford and not self.grid.has_object(position)):
             player.score -= self.grid.food_planting_cost
-            self.grid.spawn_food(position=position)
+            self.grid.spawn_object(position=position)
```

Comment:
I think this needs an `object_id`, specifically the object_id for "Food". Maybe that's a new experiment config parameter, maybe a required `plantable_object_id` when `food_planting` is enabled. Choosing an object type at random doesn't really work for this.